jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,456 @@
+    "        \n",
+    "    Returns:\n",
+    "        The according property of the element or ``None`` if the property doesn't exist.\n",
+    "    \"\"\"\n",
+    "    return next((e.value for e in element.properties.properties if e.name == property_name), None)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Retrieve Metadata with a Data Link\n",
+    "\n",
+    "A data link is the input for each __Analysis Automation procedure__ that uses a query to collect specific data items. A `data_link` contains a list of one or more elements that point to a list of files, channel groups, or channels (depending on the query result type).\n",
+    "\n",
+    "This example shows how the Metadata API accesses the `file_info` structure from the file, through the `groups`, and down to the `channels` level.\n",
+    "\n",
+    "This example calculates the absolute minimum and absolute maximum value of all channels in each group and displays these values in the report."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_links = ni_analysis_automation[\"data_links\"]\n",
+    "file_ids = [d[\"fileId\"] for d in data_links]\n",
+    "file_infos = await metadata_api.get_multiple_file_info(tdmreader.FileList(file_ids))\n",
+    "file_info = file_infos[0]\n",
+    "\n",
+    "test_file_name = get_property(file_info, \"name\")\n",
+    "program_name = get_property(file_info, \"Test~Procedure\")\n",
+    "\n",
+    "group_names = []\n",
+    "channels = []\n",
+    "formatted_properties = []\n",
+    "\n",
+    "for group in file_info.groups:\n",
+    "    group_names.append(group.name)\n",
+    "    channels.append(group.channels)\n",
+    "    max_values_of_group = []    \n",
+    "    min_values_of_group = []    \n",
+    "    mean_values_of_group = []    \n",
+    "\n",
+    "    for channel in group.channels:\n",
+    "        minimum = float(get_property(channel, \"minimum\") or \"NaN\")\n",
+    "        maximum = float(get_property(channel, \"maximum\") or \"NaN\")\n",
+    "        mean_values_of_group.append((minimum + maximum) / 2)\n",
+    "        max_values_of_group.append(maximum)\n",
+    "        min_values_of_group.append(minimum)\n",
+    "\n",
+    "        \n",
+    "    # Calculate statistical values from metadata\n",
+    "    abs_min = max(max_values_of_group)\n",
+    "    abs_max = min(max_values_of_group)\n",
+    "    abs_mean = sum(mean_values_of_group) / float(len(mean_values_of_group))\n",
+    "    formatted_properties.append(f\"Absolute Maximum: {abs_max:.3f} °C\"+\n",
+    "                                f\",Absolute Minimum: {abs_min:.3f} °C\"+\n",
+    "                                f\",Mean Value: {abs_mean:.3f} °C\")\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define Notebook Parameters.\n",
```

Comment:
For consistency with the titles of other cells, this should not have a period at the end.

Reply:
Thanks